share/doc/websiteAndDocumentation.rst

```diff
@@ -0,0 +1,123 @@
+Website and Documentation
+=========================
+
+On this page you will find all the information you need to contribute to VOTCA's
+website and documentation.
+
+The website is build with Sphinx and uses a slightly modified version of the
+Read the Docs theme. Parts of the documentation are automatically generated
+based on the source code of VOTCA and jupyter notebooks, other parts are written
+by hand. The markup language of Sphinx is reStructuredText, think of it as
+Markdown on steroids. 
+
+To start working on the website and documentation you will first need to be able
+to build it, this is discussed first. Next a brief overview of how to edit
+existing pages and adding new ones is given. Finally we discuss how to change the theme and publish the changes.
+
+Building the documentation and website locally
+----------------------------------------------
+
+To be able to see and test changes you make to the website, you will need to
+build it locally. You will first need to install and build VOTCA. Once you have
+installed VOTCA, navigate to its build folder (if you followed the install
+instructions, it will be called ``builddir``). In the build folder run 
+
+.. code-block:: bash
+
+  make doc
+
+This will probably output errors on the first few tries, read them carefully
+since they will almost always be missing python packages. Find which
+package/module is missing in the error message and simply install it with ``pip3
+install <packagename>``.
+
+A short list of packages you probably need to install manually:
+
+- cma
+- recommonmark
+- nbsphinx
+- sphinx_rtd_theme 
+
+Once the errors are gone the documentation is build and a static html website is
+generated in the ``sphinx.html`` folder within the build folder. You can view
+and test the website in any browser simply by navigating to the index.html file
+within the ``sphinx.html`` folder.
+
+Rebuilding the website after changes
+------------------------------------
+
+After you updated files you will need to rebuild the website to see your changes.
+
+If you simply changed the contents of a .rst file you can run ``make doc`` again
+to apply the changes. If you refresh the browser the changes will have taken
```

Comment:
Change this sentence to:

"you can rerun ``make doc`` to propagate the changes.``

"," after browser.